gf-server-api/src/model/Room.ts

```diff
@@ -99,25 +91,9 @@ export interface Room {
    */
   promo: string,
   /**
-   * 1 slider image URL of the room
-   */
-  slider: string,
-  /**
-   * 2 slider image URL of the room
-   */
-  slider2: string,
-  /**
-   * 3 slider image URL of the room
-   */
-  slider3: string,
-  /**
-   * 4 slider image URL of the room
-   */
-  slider4: string,
-  /**
-   * 5 slider image URL of the room
+   * slider images URL of the rooms
    */
-  slider5: string,
+  slider: string[],
```

Comment:
1. Why we can't set 6 urls?
2.  bad naming

Reply:
There were 5 slides in the model, i made an array of slides